sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -290,6 +289,32 @@ private[ui] class ExecutionPagedTable(
         }
       }
 
+    var headerNameWithTooltips: Map[String, String] = Map("ID" -> "Execution ID of the SQL query.",
+      "Description" ->
+        ("Description of the SQL query. When click on the description link," +
+          " it will navigate to query details page."),
+      "Submitted" -> "Submitted",
+      "Duration" ->
+        "Difference between start time and close time.")
+
+      if (showRunningJobs && showSucceededJobs && showFailedJobs) {
```

Comment:
We can remove these if conditions and make a single map in the first instance itself, I guess? Because, this is a map and will use the key-value only if exists in the header field.

Reply:
I am not sure we need the assert condition here. In #25723 we pass 2 sequence to list table, and access via index. Hence the size need to be same there.